src/engine/SCons/Tool/JavaCommon.py

```diff
@@ -322,12 +361,12 @@ def parseToken(self, token):
             # 'Foo$1Inner'
             # https://github.com/SCons/scons/issues/2087
             if self.outer_state.localClasses and \
-                            self.outer_state.stackBrackets[-1] > \
-                                    self.outer_state.stackBrackets[-2]+1:
+                    self.outer_state.stackBrackets[-1] > \
```

Comment:
maybe parens here to group the "a > b" part would make it more readable? and eliminate the need for that backslash-line-continuation

Reply:
What does pylint think of such? ;)

Reply:
No particular reaction one way or the other. .  Has plenty of other things to say, though...  included just for grins:

[JavaCommon-lint.txt](https://github.com/mwichmann/scons/files/3104621/JavaCommon-lint.txt)